spec/registrant_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/registrant'
+
+RSpec.describe Registrant do
+  before(:each) do
+    @registrant_1 = Registrant.new('Bruce', 18, true )
+    @registrant_2 = Registrant.new('Penny', 15 )
+  end
+
+  describe "Initialize" do
+    it "exists" do
+      expect(@registrant_1).to be_an_instance_of(Registrant)
+      expect(@registrant_2).to be_an_instance_of(Registrant)
+    end
+  end
+
+  describe "registrant data" do
+    it "checks registrant_1 data" do
+      expect(@registrant_1.name).to eq("Bruce")
+      expect(@registrant_1.age).to eq(18)
+      expect(@registrant_1.permit?).to eq(true)
+      expect(@registrant_1.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    end
+
+    it "checks registrant_2 data" do
+      expect(@registrant_2.name).to eq("Penny")
+      expect(@registrant_2.age).to eq(15)
+      expect(@registrant_2.permit?).to eq(false)
+      expect(@registrant_2.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    end
+  end
+
+  describe "earn permit method" do
+    it "earns permit" do
+      @registrant_2 = Registrant.new('Penny', 15 )
+      @registrant_2.earn_permit
+      expect(@registrant_2.permit?).to eq(true)
```

Comment:
I like to see the change happening within the test. I would have started with an assertion that .permit? was false for Penny, under line 34, then did the rest of the testing you did to make sure it all changed. Also, you don't need to do line 34 here, you did that in your before(:each) block so you can now just call @registrant_2 here and it will be that object.